tools/packaging/release/release.sh

```diff
@@ -47,6 +47,42 @@ function _check_required_env_var()
 		_die "\"$1\" environment variable is required but was not set"
 }
 
+function _next_release_version()
+{
+	local current_release=$(cat "$repo_root_dir/VERSION")
+	local current_major
+	local current_everything_else
+	local next_major
+	local next_minor
+
+	read current_major current_minor current_everything_else < <(echo $current_release | ( IFS="." ; read major minor everything_else && echo ${major:0:1} ${minor:0:1} $everything_else ))
```

Comment:
I think there is a bug in here that a double digit release with a minor update e.g. 3.10.0 will be stripped and end up as 3.2.0? and I assume we'd want to support this with monthly minor releases?

Reply:
Oh, that's a bug indeed! :-)

Reply:
Looks good to me now. Thanks!